src/status_im/multiaccounts/login/core.cljs

```diff
@@ -624,14 +625,15 @@
        login)
       (rf/merge
        cofx
-       {:db (dissoc db :goto-key-storage?)}
+       {:db                 (dissoc db :goto-key-storage?)
+        :theme/change-theme :dark}
```

Comment:
what if a user will change os theme?

Reply:
i mean i feel like this is fragile solution, we need mechanism to override components theme

Reply:
Hi, this is not an alternative to the components overrides theme feature, we still have that.

This only changes the theme for onboarding screens, as same as we initialize appearance on login.
And as onboarding always going to be dark, it doesn't matter what the system theme is.

So in a way, this only changes the theme for a single root, like
onboarding - always dark
after that - users choice

Reply:
oh thats already sounds too complex

Reply:
I don't understand what's complex here, Before this change, we had a theme like

> Onboarding - System theme
> After Onboarding - User's choice

**After this change**

> Onboarding - Dark theme
> After Onboarding - User's choice

(Its the same as always, we are just hard coding the onboarding theme)

Reply:
ok now i get it thanks :)